shared/js/ui/models/user-data.es6.js

```diff
@@ -13,7 +13,7 @@ UserData.prototype = window.$.extend({},
 
         logout () {
             this.fetch({ logout: true })
-                .then(() => this.set('loggingOut', true))
+                .then(() => this.set('userName', null))
```

Comment:
In the previous iteration this had a transient state (`loggingOut`), but now we either have the `userName` or we don't.